notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<blockquote>where sea ice does not form, land regions, and the <a href="https://blogs.egu.eu/divisions/cr/2016/10/14/image-of-the-week-the-polar-hole/" target="_blank">polar hole</a>.</blockquote><p>Sea ice does form over the polar hole (in the Arctic), we just don't observe it.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>    This mentions three separate regions that are masked:</p><p>    1. Regions where sea ice does not form</p><p>    1. Land regions</p><p>    1. Polar hole (which as you mention is not observed)</p><p>    Have updated to include it as a region that is unobserved.</p>